### PR TITLE
[4.3] Workflow code comment

### PR DESCRIPTION
**Pull request by brianteeman**
The code comments here do not make sense. I have attempted to correct that but I am not 1000% certain I have understood it correctly. Happy to correct it further.

_There is no point in having a comment in the code if it doesn't make sense_

cc @laoneo @HLeithner @obuisard

### CODE REVIEW

File: libraries/src/MVC/Model/WorkflowBehaviorTrait.php [hunk 1]
```diff
@@ -83,7 +83,7 @@ public function setUpWorkflow($extension)
         if (method_exists($this, 'getDatabase')) {
             $db = $this->getDatabase();
         } else {
-            @trigger_error('In Joomla 6.0 will the getDatabase function be mandatory.', E_USER_DEPRECATED);
+            @trigger_error('From 6.0 using the getDatabase function will be mandatory.', E_USER_DEPRECATED);
```
Comment (HLeithner) [inline libraries/src/MVC/Model/WorkflowBehaviorTrait.php]:
```suggestion
            @trigger_error('From 6.0 implementing the getDatabase method will be mandatory.', E_USER_DEPRECATED);
```

Comment (brianteeman) [reply libraries/src/MVC/Model/WorkflowBehaviorTrait.php]: thanks

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 9be303b747ad1e6583cb5950db24d6cbfea110b8<br /><br />Review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40360">issues.joomla.org/tracker/joomla-cms/40360</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 9be303b747ad1e6583cb5950db24d6cbfea110b8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40360">issues.joomla.org/tracker/joomla-cms/40360</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40360">issues.joomla.org/tracker/joomla-cms/40360</a>.</sub>

**Comment by brianteeman** [comment]
thx

**Pull request merged**